pkg/csi_driver/node_test.go

```diff
@@ -106,27 +106,28 @@ func TestNodePublishVolume(t *testing.T) {
 			},
 			expectedMount: &mount.MountPoint{Device: "/test-device", Path: testTargetPath},
 		},
-		{
-			name: "valid request with user mount options",
-			req: &csi.NodePublishVolumeRequest{
-				VolumeId:   testVolumeId,
-				TargetPath: testTargetPath,
-				VolumeCapability: &csi.VolumeCapability{
-					AccessType: &csi.VolumeCapability_Mount{
-						Mount: &csi.VolumeCapability_MountVolume{
-							MountFlags: []string{"foo", "bar"},
-						},
-					},
-					AccessMode: &csi.VolumeCapability_AccessMode{
-						Mode: csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER,
-					},
-				},
-				VolumeAttributes: testVolumeAttributes,
-			},
-			actions: []mount.FakeAction{{Action: mount.FakeActionMount}},
-			// TODO: add all mount options into FakeMounter
-			expectedMount: &mount.MountPoint{Device: testDevice, Path: testTargetPath, Type: "nfs"},
-		},
+		// TODO: add all mount options into FakeMounter
```

Comment:
why is this failing?

Reply:
FakeMounter's behavior has changed after updating the vendor files. Before the update it would check each mount option and include it in the `expectedMount` Opts field. Now it just appends any options passed to `expectedMount` Opts field. The test expects no options but FakeMounter actually returns "foo" and "bar" as options.

Reply:
I think the new behavior is correct.  The driver currently passes through all mount options to the mount command, so the expectedMount should have those mount options

Reply:
Okay will change this test to reflect that then.